plugins/source/gcp/resources/services/networkconnectivity/internal_ranges.go

```diff
@@ -0,0 +1,49 @@
+package networkconnectivity
+
+import (
+	"context"
+
+	"github.com/apache/arrow/go/v14/arrow"
+	"google.golang.org/api/networkconnectivity/v1"
+
+	"github.com/cloudquery/cloudquery/plugins/source/gcp/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func InternalRanges() *schema.Table {
+	return &schema.Table{
+		Name:        "gcp_networkconnectivity_internal_ranges",
+		Description: `https://cloud.google.com/network-connectivity/docs/reference/networkconnectivity/rest/v1/projects.locations.internalRanges/list`,
+		Resolver:    fetchInternalRanges,
+		Multiplex:   client.ProjectMultiplexEnabledServices("networkconnectivity.googleapis.com"),
+		Transform:   client.TransformWithStruct(&networkconnectivity.InternalRange{}, transformers.WithPrimaryKeys("Name")),
+		Columns: []schema.Column{
+			{
+				Name:     "project_id",
+				Type:     arrow.BinaryTypes.String,
+				Resolver: client.ResolveProject,
+			},
+		},
```

Comment:
It may be a good idea to reuse code here:
https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gcp/client/columns.go#L8
```suggestion
		Columns: schema.ColumnList{client.ProjectIDColumn(false)},
```

Reply:
done